src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClient.java

```diff
@@ -295,9 +292,13 @@ public void onBlock(org.ethereum.core.Block block, List<TransactionReceipt> rece
          */
         private boolean isSVPSpendTxReadyToSign(long currentBlockNumber, Map.Entry<Keccak256, BtcTransaction> svpSpendTx) {
             try {
+                BtcTransaction btcTx = svpSpendTx.getValue();
```

Comment:
I would use a better name, like:
```suggestion
                BtcTransaction svpSpendTxToRemoveSignatures = svpSpendTx.getValue();
```

Or maybe // svpSpendTxWithoutSignatures. Wdyt?

Reply:
Updated